presto-spi/src/main/java/com/facebook/presto/spi/statistics/TableStatistics.java

```diff
@@ -25,11 +25,16 @@
 
 public final class TableStatistics
 {
-    public static final TableStatistics EMPTY_STATISTICS = TableStatistics.builder().build();
+    private static final TableStatistics EMPTY = TableStatistics.builder().build();
 
     private final Estimate rowCount;
     private final Map<ColumnHandle, ColumnStatistics> columnStatistics;
 
+    public static TableStatistics empty()
```

Comment:
`emptyStatistics`? That way you could static import this.

Reply:
Ditto. e.g. `Optional.empty()`